covalent/_results_manager/result.py

```diff
@@ -109,71 +109,71 @@ def __str__(self):
         return show_result_str
 
     @property
-    def start_time(self):
+    def start_time(self) -> datetime:
         """
         Start time of processing the dispatch.
         """
 
         return self._start_time
 
     @property
-    def end_time(self):
+    def end_time(self) -> datetime:
         """
         End time of processing the dispatch.
         """
 
         return self._end_time
 
     @property
-    def results_dir(self):
+    def results_dir(self) -> str:
         """
         Results directory used to save this result object.
         """
 
         return self._results_dir
 
     @property
-    def lattice(self):
+    def lattice(self) -> Lattice:
```

Comment:
I don't see Lattice being imported anywhere. This is a covalent specific data type. You'd probably need to import it before using it. There might also be issues with circular imports when doing that in which case, you might have to just use `"Lattice"` instead of `Lattice`.